test/integration/nighthawk_test_server.py

```diff
@@ -283,6 +273,28 @@ def stop(self):
     os.remove(self._admin_address_path)
     return self.stopSubprocess()
 
+  # TODO(kbaichoo): migrate to a utility file.
+  @staticmethod
```

Comment:
Use of staticmethod should be avoided. Can we make this a plain function (public if need be)?

https://google.github.io/styleguide/pyguide.html#217-function-and-method-decorators

Reply:
done, plain fn in module for now.